Milestone 2 : Resolved button styling issues and use of right headers

ibrahim777764 opened the pull request:
### Milestone :Portfolio: mobile version
understood how to parse a Figma design to create a UI.
used Flexbox to place elements in the page.
used CSS Grid to place elements in the page.
Build a personal portfolio site.

ibrahimnaseer896-ui committed:
add images and update html and css file

ibrahimnaseer896-ui committed:
add images and update html and css file

ibrahimnaseer896-ui committed:
updated html and css file

ibrahim777764 commented:
please review my code and let me know if there any changes

ibrahim777764 commented:
please review my code and let me know if there any changes

medaminedev66 commented on style.css:
```diff
@@ -84,3 +84,263 @@ li {
+  color: rgb(255, 255, 255);
+  line-height: 24px;
+  bottom: 8px;
+}
```
- [ ] Let's make your button design better by removing the default border. You can use `border: none` for that.
- [ ] Let's add the button margin to your button to improve the UI design.
<img width="219" alt="image" src="https://user-images.githubusercontent.com/77190203/154450649-d9ba4a12-3825-4269-a85f-51cc8970014a.png">

medaminedev66 commented on style.css:
```diff
@@ -84,3 +84,263 @@ li {
+  border-color: #d0d9d4;
+  border-radius: 4px;
+  height: 50px;
+}
```
- [ ] Let's remove the border from this button too. It would be better to do it for all the buttons on your page.
<img width="171" alt="image" src="https://user-images.githubusercontent.com/77190203/154451091-a81d5550-4cce-4663-92a0-4579944cee62.png">

medaminedev66 commented on index.html:
```diff
@@ -65,6 +65,215 @@ <h1>
+
+
+    <section class="work">
+        <h1>My Recent Works</h1>
```
- [ ] It would be better to change the `h1` to `h2` or `h3` for this title. In general, you should only have one h1 per page, and that h1 should succinctly represent the content on that page. As long as you are willing to style with CSS, find out which level of headings (h2, h3, h4, etc) that properly represent the headings you want to use, and then style them instead.

medaminedev66 commented on index.html:
```diff
@@ -65,6 +65,215 @@ <h1>
+        </article>
+    </section>
+    <section class="about-me">
+        <h1>About me</h1>
```
- [ ] The same thing for this title, It would be better to change the `h1` to `h2` or `h3` for this title.

medaminedev66 commented on index.html:
```diff
@@ -65,6 +65,215 @@ <h1>
+        </ul>
+    </section>
+    <footer class="footer-box">
+        <h1>
+            I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+        </h1>
```
- [ ] The same thing for this title, It would be better to change the `h1` to `h2` or `h3` for this title.

medaminedev66 reviewed (changes requested):
Hi Student  😃, wave:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- ✔️The repo looks professional
- ✔️Git/GitHub flow best practices have been followed
- ✔️There are no linter errors
- ✔️Your design is good and matches the Figma design

## Required Changes ♻️
- [ ] Pull Request should have a descriptive title telling others what changes are inside. It would be better to change the title of your pull request to a descriptive title so your project is documented in a professional way.

_See the inline comments below for more required changes_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
You can also consider:
- N/V

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ibrahimnaseer896-ui committed:
updated css and html file

ibrahimnaseer896-ui committed:
updated css file

ibrahim777764 commented:
### Milestone :updated suggested required changes
**button styling** : remove borders  and give margin to all buttons
changed h1 to h3

medaminedev66 reviewed (changes requested):
Hi @ibrahim777764 , 

Great work on making the changes requested by me 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
- [ ] Pull Request should have a descriptive title telling others what changes are inside. It would be better to change the title of your pull request to a descriptive title so your project is documented in a professional way. Check out this [link ](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)to learn how to document your project professionally.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ibrahim777764 commented:
### Milestone 2 : Resolved button styling issues and use of right headers

- button styling ,removed borders and give margin to all buttons
- better header practice: changed h1 to h3

keroloussamy reviewed (approved):
Hi @ibrahim777764, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️
- Add all the requirements. ✔️

![clap](https://media0.giphy.com/media/j2GPljpvL8Lulm3d0X/giphy.gif)


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
